### PR TITLE
Add background to heading in Contact Us page

### LINKED ISSUE #98: Enhancement: Contact Us Page Map
image

Should add some sort of background to the `Contact Us` text on the map so that there is better contrast and for better readability of the title.

### PR DESCRIPTION
**Pull request by EshaanAgg**
## Description

- Added my details in `members.json`
- Add background to the heading on ContactUs page for better contrast

## Fixes

Fixes #98 by @EshaanAgg

## Types of changes

- [X] Bug/Hot fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read the **CONTRIBUTING** guidelines in repository.
- [X] My pull request has a descriptive title and is attached to an existing issue.
- [X] I haven't repeated the code and it follows established code style of repository.
- [X] My branch is up-to-date with upstream branch.
- [X] All new and existing tests passed.

### CODE REVIEW

File: pages/contact/index.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <main class="text-center text-light-50">
     <div id="head-map" class="flex justify-center items-center">
       <Map class="w-full" />
-      <span class="absolute text-light-50 text-6xl xl:text-8xl font-bold font-display">Contact Us</span>
+        <span class="map-title absolute text-light-50 text-6xl xl:text-8xl font-bold font-display">Contact Us</span>
```
Comment (IamEzio) [inline pages/contact/index.vue:5]: Add other font-size classes to this span so the heading is responsive for small screen sizes as well. Currently, for mobile screens, the heading (and the new bg added) occupies much of the space making it difficult to appreciate the map behind.

File: pages/contact/index.vue [hunk 2]
```diff
@@ -210,4 +210,9 @@ ul li a:hover {
   background-color: rgba(255, 255, 255, 0.979);
   box-shadow: 0px 0px 18px rgba(116, 227, 235, 0.959);
 }
+.map-title{
```
Comment (IamEzio) [inline pages/contact/index.vue:213]: Use windicss classes wherever applicable to maintain consistency. We normally avoid css if there is a windi css class to do the same.

**Review by IamEzio — CHANGES_REQUESTED** [review]

File: content/members/members.json [hunk 3]
```diff
@@ -411,5 +411,12 @@
     "github": "https://github.com/PaarthAgarwal",
     "link": "https://www.linkedin.com/in/paarth-agarwal-2291911b8/",
     "year": 2020
+  },
```
Comment (IamEzio) [inline content/members/members.json:414]: For now, remove this change as we'll update new members' data later on.

Comment (EshaanAgg) [reply content/members/members.json]: Removed in the latest commit

File: pages/contact/index.vue (hunk 2 above)
Comment (EshaanAgg) [reply pages/contact/index.vue:218]: Shifted `padding` and `border-radius` properties to windicss. Now using `map-title` class only to set the background of the text.

File: pages/contact/index.vue (hunk 1 above)
Comment (EshaanAgg) [reply pages/contact/index.vue]: Added responsive classes in the latest commit.

File: pages/contact/index.vue [hunk 4]
```diff
@@ -2,7 +2,12 @@
   <main class="text-center text-light-50">
     <div id="head-map" class="flex justify-center items-center">
       <Map class="w-full" />
-      <span class="absolute text-light-50 text-6xl xl:text-8xl font-bold font-display">Contact Us</span>
+      <span class="map-title rounded-full absolute text-light-50 font-bold font-display
+      text-8xl p-4
+      <lg:(text-6xl p-4)
+      <md:(text-4xl p-4)
+      <sm:(text-2xl p-2)
```
Comment (IamEzio) [inline pages/contact/index.vue:9]:
```suggestion
```

**Review by IamEzio — APPROVED** [review]
Thanks @EshaanAgg. Just a final nit.

**Pull request merged**